### PR TITLE
[icons] feat: new icons "antenna", "tank"

### PR DESCRIPTION
**Pull request by emmalangfitt**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:

Add 16px and 20px antenna and tank icons 

#### Reviewers should focus on:

Correct files changed, icons render correctly

#### Screenshot

Screen Shot 2021-08-11 at 10 01 57 AM
Screen Shot 2021-08-11 at 10 01 51 AM

### CODE REVIEW

File: resources/icons/16px/antenna.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (aycai) [inline resources/icons/16px/antenna.svg]: nit: `fill="none"` isn't needed (same applies to the other files)

File: resources/icons/16px/antenna.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...78688 3.23997 10.5771 3.71524C11.3673 4.19051 12.0106 4.87504 12.4361 5.69315C12.8615 6.51126 13.0524 7.43108 12.9877 8.35092C12.9625 8.70916 12.8989 9.06198 12.7989 9.40358L12.7958 9.41419C12.733 9.62732 12.656 9.83605 12.5653 10.039C12.3401 10.5432 11.705 10.6451 11.2555 10.3243C10.9175 10.0829 10.7859 9.66952 10.8078 9.25404C10.8113 9.18833 10.8186 9.12257 10.8296 9.05743C10.8414 8.98679 10.8576 8.91688 10.8777 8.84856C10.9388 8.6416 10.9776 8.42774 10.9928 8.21055C11.0317 7.6586 10.9171 7.10666 10.6619 6.61576C10.4066 6.12485 10.0205 5.7141 9.54637 5.42891C9.07221 5.14373 8.52843 4.99522 7.97514 4.9998C7.42184 5.00439 6.88059 5.16189 6.41123 5.4549C5.94187 5.7479 5.56267 6.16499 5.31557 6.66006C5.06846 7.15513 4.96307 7.70889 5.01106 8.26012C5.02994 8.47703 5.07228 8.69022 5.13673 8.89614ZM7.99999 6.99989C8.55228 6.99989 8.99999 7.44761 8.99999 7.99989V11.5857L11.7071 14.2928C12.0976 14.6833 12.0976 15.3165 11.7071 15.707C11.3166 16.0975 10.6834 16.0975 10.2929 15.707L7.99999 13.4141L5.7071 15.707C5.31658 16.0975 4.68341 16.0975 4.29289 15.707C3.90236 15.3165 3.90236 14.6833 4.29289 14.2928L6.99999 11.5857V7.99989C6.99999 7.44761 7.44771 6.99989 7.99999 6.99989Z" fill="black"/>
```
Comment (aycai) [inline resources/icons/16px/antenna.svg]: nit: `fill="black"` (at the end) isn't needed (same applies to the other files)

**Comment by aycai** [comment]
Designs look good! Left a few nits

File: resources/icons/16px/antenna.svg (hunk 1 above)
Comment (adidahiya) [reply resources/icons/16px/antenna.svg]:
```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

File: resources/icons/16px/antenna.svg (hunk 2 above)
Comment (adidahiya) [reply resources/icons/16px/antenna.svg]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M2.67261 10.7577C2.68007 10.7722 2.68724 10.7867 2.69412 10.8013C2.70553 10.8256 2.71613 10.8501 2.72595 10.8749C2.74125 10.9135 2.75461 10.9528 2.76602 10.9924C2.89304 11.4337 2.77766 11.9239 2.40414 12.2045C1.96261 12.5363 1.32909 12.4501 1.05497 11.9707C0.492513 10.9869 0.143532 9.89133 0.035771 8.75563C-0.103594 7.28685 0.165927 5.80813 0.814535 4.48297C1.46314 3.15781 2.46557 2.03783 3.711 1.24684C4.95642 0.455851 6.39632 0.0246739 7.87151 0.000977833C9.3467 -0.0227182 10.7997 0.36199 12.0699 1.11257C13.3401 1.86315 14.378 2.95036 15.0688 4.25401C15.7596 5.55765 16.0765 7.02695 15.9844 8.49945C15.9132 9.63803 15.5995 10.7442 15.069 11.7456C14.8104 12.2336 14.18 12.34 13.728 12.0226C13.3457 11.7541 13.2146 11.2679 13.3274 10.8228C13.3375 10.7828 13.3496 10.7431 13.3637 10.704C13.3727 10.6789 13.3825 10.6541 13.3931 10.6295C13.3995 10.6146 13.4062 10.5999 13.4132 10.5852C13.743 9.89464 13.939 9.14444 13.9872 8.3745C14.0563 7.27033 13.8186 6.16857 13.3006 5.19102C12.7826 4.21347 12.0043 3.39821 11.0519 2.83538C10.0994 2.27255 9.00984 1.98407 7.90365 2.00184C6.79747 2.01961 5.71774 2.34293 4.78385 2.93606C3.84995 3.52919 3.09827 4.36903 2.61191 5.36271C2.12554 6.35639 1.92344 7.46522 2.02794 8.5666C2.10082 9.3346 2.32079 10.0781 2.67261 10.7577ZM5.13673 8.89614C5.15801 8.96412 5.17531 9.03375 5.18834 9.10418C5.20037 9.16913 5.20877 9.23476 5.21333 9.3004C5.24215 9.71546 5.11738 10.131 4.78343 10.3779C4.33938 10.7062 3.70261 10.6148 3.46908 10.1144C3.3751 9.91297 3.29465 9.70554 3.22828 9.49349L3.22499 9.48293C3.11942 9.14303 3.04998 8.79132 3.01884 8.43354C2.93887 7.5149 3.1145 6.59204 3.52631 5.76699C3.93811 4.94194 4.57006 4.24684 5.35227 3.75854C6.13448 3.27024 7.03648 3.00776 7.95856 3.00012C8.88064 2.99248 9.78688 3.23997 10.5771 3.71524C11.3673 4.19051 12.0106 4.87504 12.4361 5.69315C12.8615 6.51126 13.0524 7.43108 12.9877 8.35092C12.9625 8.70916 12.8989 9.06198 12.7989 9.40358L12.7958 9.41419C12.733 9.62732 12.656 9.83605 12.5653 10.039C12.3401 10.5432 11.705 10.6451 11.2555 10.3243C10.9175 10.0829 10.7859 9.66952 10.8078 9.25404C10.8113 9.18833 10.8186 9.12257 10.8296 9.05743C10.8414 8.98679 10.8576 8.91688 10.8777 8.84856C10.9388 8.6416 10.9776 8.42774 10.9928 8.21055C11.0317 7.6586 10.9171 7.10666 10.6619 6.61576C10.4066 6.12485 10.0205 5.7141 9.54637 5.42891C9.07221 5.14373 8.52843 4.99522 7.97514 4.9998C7.42184 5.00439 6.88059 5.16189 6.41123 5.4549C5.94187 5.7479 5.56267 6.16499 5.31557 6.66006C5.06846 7.15513 4.96307 7.70889 5.01106 8.26012C5.02994 8.47703 5.07228 8.69022 5.13673 8.89614ZM7.99999 6.99989C8.55228 6.99989 8.99999 7.44761 8.99999 7.99989V11.5857L11.7071 14.2928C12.0976 14.6833 12.0976 15.3165 11.7071 15.707C11.3166 16.0975 10.6834 16.0975 10.2929 15.707L7.99999 13.4141L5.7071 15.707C5.31658 16.0975 4.68341 16.0975 4.29289 15.707C3.90236 15.3165 3.90236 14.6833 4.29289 14.2928L6.99999 11.5857V7.99989C6.99999 7.44761 7.44771 6.99989 7.99999 6.99989Z"/>
```

File: resources/icons/16px/tank.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (adidahiya) [inline resources/icons/16px/tank.svg]:
```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

File: resources/icons/16px/tank.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M3.7 3.4C3.88885 3.14819 4.18524 3 4.5 3H7H9.58579C9.851 3 10.1054 3.10536 10.2929 3.29289L11 4V4.00003H14C14.5523 4.00003 15 4.44775 15 5.00003C15 5.55232 14.5523 6.00003 14 6.00003H11V6.99998H13.5C14.8807 6.99998 16 8.11927 16 9.49998C16 10.8807 14.8807 12 13.5 12H2.5C1.11929 12 0 10.8807 0 9.49998C0 8.11927 1.11929 6.99998 2.5 6.99998H3V4.66667C3 4.4503 3.07018 4.23976 3.2 4.06667L3.7 3.4ZM2.5 8.99998H13.5C13.7761 8.99998 14 9.22384 14 9.49998C14 9.77613 13.7761 9.99998 13.5 9.99998H2.5C2.22386 9.99998 2 9.77613 2 9.49998C2 9.22384 2.22386 8.99998 2.5 8.99998Z" fill="black"/>
```
Comment (adidahiya) [inline resources/icons/16px/tank.svg]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M3.7 3.4C3.88885 3.14819 4.18524 3 4.5 3H7H9.58579C9.851 3 10.1054 3.10536 10.2929 3.29289L11 4V4.00003H14C14.5523 4.00003 15 4.44775 15 5.00003C15 5.55232 14.5523 6.00003 14 6.00003H11V6.99998H13.5C14.8807 6.99998 16 8.11927 16 9.49998C16 10.8807 14.8807 12 13.5 12H2.5C1.11929 12 0 10.8807 0 9.49998C0 8.11927 1.11929 6.99998 2.5 6.99998H3V4.66667C3 4.4503 3.07018 4.23976 3.2 4.06667L3.7 3.4ZM2.5 8.99998H13.5C13.7761 8.99998 14 9.22384 14 9.49998C14 9.77613 13.7761 9.99998 13.5 9.99998H2.5C2.22386 9.99998 2 9.77613 2 9.49998C2 9.22384 2.22386 8.99998 2.5 8.99998Z"/>
```

File: resources/icons/20px/antenna.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (adidahiya) [inline resources/icons/20px/antenna.svg]:
```suggestion
<svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
```

File: resources/icons/20px/antenna.svg [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
....21358 16.0629 9.31736 15.9852 10.4212C15.9459 10.9798 15.8288 11.5273 15.6394 12.0487C15.6221 12.0963 15.6042 12.1436 15.5858 12.1907C15.5265 12.3419 15.461 12.4908 15.3895 12.6369C15.1492 13.1281 14.5224 13.2288 14.0774 12.9111C13.7232 12.6582 13.5942 12.2159 13.657 11.7893C13.6677 11.7162 13.6841 11.6437 13.7059 11.5726C13.7167 11.5372 13.7289 11.5023 13.7425 11.4678C13.8906 11.0902 13.9813 10.6906 14.0101 10.2822C14.0621 9.54263 13.9086 8.8031 13.5666 8.14534C13.2246 7.48758 12.7073 6.93721 12.072 6.5551C11.4367 6.17298 10.708 5.97399 9.96669 5.98014C9.22534 5.98628 8.50012 6.19732 7.87123 6.58991C7.24233 6.98251 6.73425 7.54137 6.40316 8.20471C6.07206 8.86804 5.93086 9.61002 5.99515 10.3486C6.03065 10.7565 6.12805 11.1545 6.28238 11.5296ZM10.9937 14.1127C10.9979 14.0757 11 14.0381 11 14V9.99999C11 9.44771 10.5523 8.99999 10 8.99999C9.44771 8.99999 9 9.44771 9 9.99999V14C9 14.0381 9.00213 14.0757 9.00628 14.1127L5.2526 18.3356C4.88568 18.7484 4.92286 19.3805 5.33564 19.7474C5.74843 20.1143 6.3805 20.0771 6.74742 19.6644L9.99999 16.0052L13.2524 19.6643C13.6193 20.0771 14.2513 20.1143 14.6641 19.7474C15.0769 19.3805 15.1141 18.7484 14.7472 18.3357L10.9937 14.1127Z" fill="black"/>
```
Comment (adidahiya) [inline resources/icons/20px/antenna.svg]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M2.01026 10.7581C2.11759 11.8893 2.46349 12.9809 3.0207 13.9623L3.04069 13.9973C3.07361 14.0547 3.10133 14.1141 3.12386 14.1748C3.28719 14.6153 3.17755 15.1262 2.79451 15.414C2.35863 15.7415 1.7351 15.6559 1.45215 15.1898C1.41011 15.1206 1.36894 15.0509 1.32864 14.9807C1.2955 14.923 1.26295 14.865 1.23099 14.8067C0.579306 13.6178 0.173726 12.3042 0.0447181 10.9446C-0.129488 9.10863 0.207413 7.26024 1.01817 5.60379C1.82893 3.94733 3.08197 2.54735 4.63875 1.55862C6.19553 0.569882 7.99541 0.0309102 9.83939 0.00129014C11.6834 -0.0283299 13.4996 0.452555 15.0874 1.39078C16.6751 2.329 17.9725 3.68802 18.836 5.31758C19.6995 6.94713 20.0956 8.78375 19.9805 10.6244C19.8952 11.9874 19.532 13.3134 18.9189 14.5226C18.8888 14.5819 18.8581 14.6409 18.8269 14.6996C18.7888 14.7711 18.7499 14.8421 18.7101 14.9126C18.4423 15.3875 17.8218 15.4931 17.3757 15.1798C16.9836 14.9045 16.8576 14.3973 17.0067 13.9518C17.0273 13.8904 17.0531 13.8301 17.0841 13.7718L17.103 13.7361C17.6284 12.7373 17.939 11.6352 18.01 10.5011C18.1024 9.02388 17.7845 7.54988 17.0915 6.24206C16.3984 4.93424 15.3572 3.84354 14.0829 3.09055C12.8087 2.33757 11.351 1.95163 9.8711 1.9754C8.39119 1.99917 6.94667 2.43173 5.69726 3.22525C4.44784 4.01878 3.4422 5.14235 2.79151 6.47176C2.14083 7.80117 1.87044 9.28462 2.01026 10.7581ZM6.28238 11.5296C6.29648 11.5639 6.30925 11.5986 6.32069 11.6338C6.34368 11.7045 6.36124 11.7768 6.3732 11.8496C6.44304 12.2752 6.32135 12.7196 5.97137 12.9783C5.53171 13.3033 4.90336 13.213 4.65496 12.7259C4.58105 12.581 4.51312 12.4332 4.45131 12.283C4.43206 12.2362 4.4134 12.1892 4.39534 12.1419C4.19731 11.6237 4.07117 11.0782 4.02261 10.5203C3.92665 9.41794 4.13741 8.31051 4.63157 7.32046C5.12574 6.3304 5.88407 5.49628 6.82272 4.91032C7.76137 4.32435 8.84378 4.00938 9.95028 4.00021C11.0568 3.99104 12.1443 4.28803 13.0925 4.85835C14.0407 5.42868 14.8128 6.25011 15.3233 7.23185C15.8338 8.21358 16.0629 9.31736 15.9852 10.4212C15.9459 10.9798 15.8288 11.5273 15.6394 12.0487C15.6221 12.0963 15.6042 12.1436 15.5858 12.1907C15.5265 12.3419 15.461 12.4908 15.3895 12.6369C15.1492 13.1281 14.5224 13.2288 14.0774 12.9111C13.7232 12.6582 13.5942 12.2159 13.657 11.7893C13.6677 11.7162 13.6841 11.6437 13.7059 11.5726C13.7167 11.5372 13.7289 11.5023 13.7425 11.4678C13.8906 11.0902 13.9813 10.6906 14.0101 10.2822C14.0621 9.54263 13.9086 8.8031 13.5666 8.14534C13.2246 7.48758 12.7073 6.93721 12.072 6.5551C11.4367 6.17298 10.708 5.97399 9.96669 5.98014C9.22534 5.98628 8.50012 6.19732 7.87123 6.58991C7.24233 6.98251 6.73425 7.54137 6.40316 8.20471C6.07206 8.86804 5.93086 9.61002 5.99515 10.3486C6.03065 10.7565 6.12805 11.1545 6.28238 11.5296ZM10.9937 14.1127C10.9979 14.0757 11 14.0381 11 14V9.99999C11 9.44771 10.5523 8.99999 10 8.99999C9.44771 8.99999 9 9.44771 9 9.99999V14C9 14.0381 9.00213 14.0757 9.00628 14.1127L5.2526 18.3356C4.88568 18.7484 4.92286 19.3805 5.33564 19.7474C5.74843 20.1143 6.3805 20.0771 6.74742 19.6644L9.99999 16.0052L13.2524 19.6643C13.6193 20.0771 14.2513 20.1143 14.6641 19.7474C15.0769 19.3805 15.1141 18.7484 14.7472 18.3357L10.9937 14.1127Z"/>
```

File: resources/icons/20px/tank.svg [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (adidahiya) [inline resources/icons/20px/tank.svg]:
```suggestion
<svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
```

File: resources/icons/20px/tank.svg [hunk 8]
```diff
@@ -0,0 +1,3 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M3.95625 4.47C4.13899 4.17762 4.45946 4 4.80425 4H8H11.1958C11.5405 4 11.861 4.17762 12.0438 4.47L13 5.99999H18C18.5523 5.99999 19 6.44772 19 7C19 7.55228 18.5523 7.99999 18 7.99999H13V8.99998H17C18.6569 8.99998 20 10.3431 20 12C20 13.6568 18.6569 15 17 15H3C1.34315 15 0 13.6568 0 12C0 10.3431 1.34315 8.99998 3 8.99998L3 6.2868C3 6.09938 3.05267 5.91573 3.152 5.7568L3.95625 4.47ZM3 11H17C17.5523 11 18 11.4477 18 12C18 12.5523 17.5523 13 17 13H3C2.44772 13 2 12.5523 2 12C2 11.4477 2.44772 11 3 11Z" fill="black"/>
```
Comment (adidahiya) [inline resources/icons/20px/tank.svg]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M3.95625 4.47C4.13899 4.17762 4.45946 4 4.80425 4H8H11.1958C11.5405 4 11.861 4.17762 12.0438 4.47L13 5.99999H18C18.5523 5.99999 19 6.44772 19 7C19 7.55228 18.5523 7.99999 18 7.99999H13V8.99998H17C18.6569 8.99998 20 10.3431 20 12C20 13.6568 18.6569 15 17 15H3C1.34315 15 0 13.6568 0 12C0 10.3431 1.34315 8.99998 3 8.99998L3 6.2868C3 6.09938 3.05267 5.91573 3.152 5.7568L3.95625 4.47ZM3 11H17C17.5523 11 18 11.4477 18 12C18 12.5523 17.5523 13 17 13H3C2.44772 13 2 12.5523 2 12C2 11.4477 2.44772 11 3 11Z"/>
```

**Comment by adidahiya** [comment]
image

@aycai top of the 16px tank and the bottom of the 20px tank look a little blurry, but this is fine for a first version of the icon

**Comment by adidahiya** [comment]
image


antenna looks great!

**Comment by aycai** [comment]
> image
> 
> @aycai top of the 16px tank and the bottom of the 20px tank look a little blurry, but this is fine for a first version of the icon

@adidahiya Strange, the vector edges line up to the pixel when I check the svg in app

**Pull request merged**